### PR TITLE
enable addButton when machine wide source lost focus

### PR DESCRIPTION
**Pull request by zhili1208**
@rrelyea

### CODE REVIEW

File: src/NuGet.Clients/Options/PackageSourcesOptionsControl.cs [hunk 1]
```diff
@@ -95,15 +95,22 @@ private void UpdateUI()
...
+            else
             {
                 // THIS BLOCK corresponds to PackageSourcesListBox
 
                 MoveUpButton.Enabled = selectedSource != null && PackageSourcesListBox.SelectedIndex > 0;
                 MoveDownButton.Enabled = selectedSource != null && PackageSourcesListBox.SelectedIndex < PackageSourcesListBox.Items.Count - 1;
 
-                // do not allow deleting the official NuGet source
-                bool allowEditing = selectedSource != null && !selectedSource.IsOfficial;
+                bool allowEditing = selectedSource != null;
```
Comment (alpaix) [inline src/NuGet.Clients/Options/PackageSourcesOptionsControl.cs:116]: Is this change intentional? Looks unrelated to the original issue.

**Comment by alpaix** [comment]
:shipit:

File: src/NuGet.Clients/Options/PackageSourcesOptionsControl.cs (hunk 1 above)
Comment (zhili1208) [inline src/NuGet.Clients/Options/PackageSourcesOptionsControl.cs:116]: Yes, it's intentional and it's unrelated to the issue, this is just old design, it's already changed, so did clean up here

File: src/NuGet.Clients/Options/PackageSourcesOptionsControl.cs [hunk 2]
```diff
@@ -95,15 +95,22 @@ private void UpdateUI()
             var selectedSource = (Configuration.PackageSource)PackageSourcesListBox.SelectedItem;
             var selectedMachineSource = (Configuration.PackageSource)MachineWidePackageSourcesListBox.SelectedItem;
 
-            if (selectedSource != null)
+            if (selectedMachineSource != null)
+            {
+                // THIS BLOCK corresponds to MachineWidePackageSourcesListBox
+
+                addButton.Enabled = removeButton.Enabled = MoveUpButton.Enabled = MoveDownButton.Enabled = BrowseButton.Enabled = updateButton.Enabled = false;
```
Comment (yishaigalatzer) [inline src/NuGet.Clients/Options/PackageSourcesOptionsControl.cs]: write each assignment in its own line

File: src/NuGet.Clients/Options/PackageSourcesOptionsControl.cs [hunk 3]
```diff
@@ -95,15 +95,22 @@ private void UpdateUI()
             var selectedSource = (Configuration.PackageSource)PackageSourcesListBox.SelectedItem;
             var selectedMachineSource = (Configuration.PackageSource)MachineWidePackageSourcesListBox.SelectedItem;
 
-            if (selectedSource != null)
+            if (selectedMachineSource != null)
+            {
+                // THIS BLOCK corresponds to MachineWidePackageSourcesListBox
```
Comment (yishaigalatzer) [inline src/NuGet.Clients/Options/PackageSourcesOptionsControl.cs]: remove all CAPS and space after the comment

File: src/NuGet.Clients/Options/PackageSourcesOptionsControl.cs (hunk 1 above)
Comment (yishaigalatzer) [inline src/NuGet.Clients/Options/PackageSourcesOptionsControl.cs:116]: that looks good to me
